docs/changelog.rst

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Please note that the changes before version 1.10.0 have not been documented.
 
+v3.0.0
+----------
+Changed
+
+- Tracking also status codes
+
```

Comment:
@bogdanp05 @FlyingBird95 @thejager -- anything else we should be adding here?

Reply:
Changes:

- Display times as numbers to make them sortable 7ebfef6
- Add leading slash to blueprint paths b370407
- Added status codes with corresponding views a8e84d9

Reply:
Cool. Updated file correspondingly.